### ISSUE TITLE
Hidden services test fails on Windows

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#5116]
```
======================================================================
FAIL: Test the hidden services overlay by constructing an end-to-end circuit and downloading a torrent over it
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\users\heteoma\appdata\local\programs\python\python37\lib\site-packages\asynctest\case.py", line 297, in run
    self._run_test_method(testMethod)
  File "c:\users\heteoma\appdata\local\programs\python\python37\lib\site-packages\asynctest\case.py", line 354, in _run_test_method
    self.loop.run_until_complete(result)
  File "c:\users\heteoma\appdata\local\programs\python\python37\lib\site-packages\asynctest\case.py", line 224, in wrapper
    return method(*args, **kwargs)
  File "c:\users\heteoma\appdata\local\programs\python\python37\lib\asyncio\base_events.py", line 579, in run_until_complete
    return future.result()
  File "C:\workspace\jenkins2\workspace\GH_Tribler_PR_Tests\PR_async_win64\tribler\src\tribler-core\tribler_core\tests\tools\tools.py", line 12, in wrapper
    await wait_for(coro(*args, **kwargs), timeout)
  File "c:\users\heteoma\appdata\local\programs\python\python37\lib\asyncio\tasks.py", line 442, in wait_for
    return fut.result()
  File "C:\workspace\jenkins2\workspace\GH_Tribler_PR_Tests\PR_async_win64\tribler\src\tribler-core\tribler_core\modules\tunnel\test_full_session\test_hidden_services.py", line 25, in test_hidden_services
    self.assertEqual(7, len(c.get_peers()))
AssertionError: 7 != 0
-------------------- >> begin captured logging << --------------------
```

**Comment by devos50** [comment]
This test seems to work now on windows:

```
Test the hidden services overlay by constructing an end-to-end circuit and downloading a torrent over it ... ok
```

**Comment by devos50** [comment]
Potentially fixed by #5279 (just gave the `deliver_messages` a higher timeout).
